pkgs/development/libraries/libint/default.nix

```diff
@@ -48,29 +50,54 @@ stdenv.mkDerivation rec {
     "--with-g12dkh-max-am=5"
     "--with-g12dkh-opt-am=3"
     "--enable-contracted-ints"
+    "--enable-fortran"
     "--enable-shared"
    ] ++ lib.optional enableFMA "--enable-fma";
+   */
+  configureFlags = [
+    "--with-eri-max-am=1"
+    "--enable-shared"
+    "--enable-fortran"
+  ] ++ lib.optional enableFMA "--enable-fma";
+
+  cmakeFlags = [
+    "-DENABLE_FORTRAN=ON"
+
+    # Default is just "double", but SSE2 is available on all x86_64 CPUs.
+    "-DLIBINT2_REALTYPE=${if enableAVX
+      then "libint2::simd::VectorAVXDouble"
+      else "libint2::simd::VectorSSEDouble"}"
+  ];
 
   preConfigure = ''
     ./autogen.sh
+    ./configure --prefix=$(pwd) ${builtins.toString configureFlags}
+    make export -j $NIX_BUILD_CORES
+    tar -xf libint-${version}.tgz
```

Comment:
At this point, I can see how splitting it up into two derivations could make sense:
1. Each of the derivations would be less hacky (?)
2. The AVX optimizations do not affect the code generation phase (?) and thus an optimized would be cheaper, when it only needs to recompile the generated code?
3. Would two derivations be more simple and readable?

Reply:
Yep, I think two are better. Maybe something like this:
```nix
let codeGen = stdenv.mkDerivation { .. };
    codeComp = stdenv.mkDerivation { .. };
in codeComp
```

I guess "yes" to all three points. But no idea what the Fortran problem is; I couldn't find anything similar in the libint issues and PRs. I will give it another try with 2 separate derivations.